EXILED/Exiled.Events/Patches/Fixes/KillPlayer.cs

```diff
@@ -1,51 +0,0 @@
-// -----------------------------------------------------------------------
-// <copyright file="KillPlayer.cs" company="ExMod Team">
-// Copyright (c) ExMod Team. All rights reserved.
-// Licensed under the CC BY-SA 3.0 license.
-// </copyright>
-// -----------------------------------------------------------------------
-
-namespace Exiled.Events.Patches.Fixes
-{
-#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
-#pragma warning disable CS0612 // Type or member is obsolete
-#pragma warning disable CS0618 // Type or member is obsolete
-
-    using API.Features;
-    using API.Features.DamageHandlers;
-
-    using EventArgs.Player;
-
-    using HarmonyLib;
-
-    using Mirror;
-
-    using PlayerStatsSystem;
-
-    using DamageHandlerBase = PlayerStatsSystem.DamageHandlerBase;
-
-    /// <summary>
-    /// Prefix of KillPlayer action.
-    /// </summary>
-    [HarmonyPatch(typeof(PlayerStats), nameof(PlayerStats.KillPlayer))]
-    internal class KillPlayer
-    {
-        private static void Prefix(PlayerStats __instance, ref DamageHandlerBase handler)
-        {
-            if (!DamageHandlers.IdsByTypeHash.ContainsKey(handler.GetType().FullName.GetStableHashCode()))
-            {
-                if (handler is GenericDamageHandler exiledHandler)
-                {
-                    handler = exiledHandler.Base;
-                }
```

Comment:
i could be wrong but i think if we removed this it's would break the DamageHandler API

Reply:
not sure in that

Reply:
Should I revert this patch?
@louis1706 @VALERA771

Reply:
Do not remove the patch, but the event call